docs/LibraryEvolution.rst

```diff
@@ -145,14 +145,30 @@ The following changes are permitted:
   body, not the labels that are part of the function's full name).
 - Reordering generic requirements (but not the generic parameters themselves).
 - Adding a default argument expression to a parameter.
-- Changing or removing a default argument is a `binary-compatible
-  source-breaking change`.
+- Adding, changing, reordering, or removing property wrappers that either are 
+  implementation-detail (``@ImplementationDetail``) or in a composition where
```

Comment:
Sorry, what is this `@ImplementationDetail` wrapper? Is this a new/upcoming feature?

Reply:
I don't think that's a real wrapper, and I don't think including it in this bullet point adds any clarity.

Reply:
OK, I'll remove it.